dockerfiles/docker-compose.yml

```
@@ -162,3 +161,18 @@ services:
     networks:
       readthedocs:
     command: ["server", "--console-address", ":9009", "/data"]
+
+  createbuckets:
+    image: minio/mc
+    volumes:
+      - ${PWD}/common/dockerfiles/entrypoints/createbuckets.sh:/createbuckets.sh
+      - ${PWD}/common/dockerfiles/entrypoints/wait-for-it.sh:/wait-for-it.sh
```

```suggestion
      - ${PWD}/common/dockerfiles/entrypoints/createbuckets.sh:/usr/src/app/docker/createbuckets.sh
      - ${PWD}/common/dockerfiles/entrypoints/wait-for-it.sh:/usr/src/app/docker/wait-for-it.sh
```

To keep consistency with the other containers.

---

I'd prefer spinning up a container that's made only to run the `mc` client, rather than spinning up a container that has a different purpose. Don't you think that's better?

It starts and exits in parallel with everything else and nothing depends on it, so it's not really slowing anything down.

---

Yes. I think it's better to use the `storage` container to create the buckets the first time it's called.

I understand your point, but I think it's better to keep things simple in this case. We are already doing this on `web`, for example, to run migrations, create super users, collectdata, etc. So, I prefer to follow the same pattern here as well.

---

Ohh! You are right! We can't do it from the `storage` container. However, we could do it from `web` in case it's easy to install `mc` there.

Taking a look at the docs, it seems it's pretty simple. Just downloading a binary file: https://docs.min.io/docs/minio-client-quickstart-guide.html (section GNU/Linux)

---

Trying out the `mc` client in the `web` container now :heart:

---

It works and has been add to the PR now :tada: